src/Http/Place/CalendarValidationRequestBodyParser.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Place;
```

Comment:
I would suggest to already put this in an `Offer` namespace instead. But only supporting `periodic` and `permanent` for now is fine. Then we can also add support for `single` and `multiple` later.

Places don't support `single` and `multiple`, but that is handled by the schema already. And then we don't need to have 2 `CalendarValidationRequestBodyParser` classes.

Reply:
Yes, the `Offer` namespace is the long-term location.